src/analytics/events.js

```diff
@@ -55,6 +55,22 @@ export function jobSearch(filters) {
   };
 }
 
+/**
+ * Returns an job search event object
+ * @param {Object|Map} filters
+ * @param int numResults
+ * @return {Object}
+ */
+export function jobSearchResults(filters, numResults) {
+  return {
+    category: 'jobs',
+    action: 'search-results',
+    ...searchDimensions(filters),
+    dimension8: numResults,                      // search:numResults
+    dimension9: numResults === 0 ? false : true, // search:hasResults
```

Comment:
Could this ever be anything other than an int? Maybe better to test `numResults > 0`?

Reply:
Good point, updated